src/TurbulenceClosures/turbulence_closure_diagnostics.jl

```diff
@@ -23,102 +31,29 @@ maximum_numeric_diffusivity(::Nothing) = 0
 # As the name suggests, we give up in the case of a function diffusivity
 maximum_numeric_diffusivity(κ::Function) = 0
 
-function cell_diffusion_timescale(closure::IsotropicDiffusivity, diffusivities, grid)
-    Δ = min_Δxyz(grid)
+function cell_diffusion_timescale(closure::ScalarDiffusivity{TD, Dir}, diffusivities, grid) where {TD, Dir}
+    Δ = min_Δxyz(grid, Dir)
```

Comment:
```suggestion
    Δ = min_Δxyz(grid, direction(closure))
```

This will make it easy to change the position of the type parameter later if we need to.

Reply:
("Only use type parameters for dispatch" - YASGuide)